giraf/giraf.go

```diff
@@ -30,23 +32,28 @@ type Giraf struct {
 	// An example would be "BZ:i:4000
 }
 
+// Path describes an alignment using a traversal slice of Node Ids as well as a start and end coordinate
 type Path struct {
 	TStart int      // The path starts on the TStart base (0-based, closed) of Nodes[0]
 	Nodes  []uint32 // The node Id/Index of all the nodes in the path
 	TEnd   int      // The path ends on the TEnd base (0-based, open) of Nodes[len(Nodes)-1]
 }
 
+// Note is a scruct containing additional information from the graph alignment
 type Note struct {
-	Tag   string
-	Type  rune
+	Tag   []byte
```

Comment:
as long as you are modifying the tag, why not make it type `[2]byte` instead of a `[]byte`

Reply:
Yes i am thinking about it. I see a few potential problems coming up once we move towards that. I had thought []byte was a good middle ground, but let me turn it over in my head.